resources/pvc.py

```diff
@@ -71,21 +71,21 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_all_pvcs():
+def delete_pvcs(pvc_name):
```

Comment:
please verify no one else is using this function as you changed its signature

Reply:
Checked no one else using this function